scene/3d/bone_attachment_3d.cpp

```diff
@@ -32,7 +32,15 @@
 
 void BoneAttachment3D::_validate_property(PropertyInfo &property) const {
 	if (property.name == "bone_name") {
-		Skeleton3D *parent = Object::cast_to<Skeleton3D>(get_parent());
+		// Because it is a constant function, we cannot use the _get_skeleton_3d function.
+		Skeleton3D *parent = nullptr;
```

Comment:
We use the `parent` only to fetch data from `const` functions:
- get_bone_count
- get_bone_name

Consider to make this `const`, will improve the function coherency.